partiql-parser/src/test/kotlin/org/partiql/parser/internal/PartiQLParserDDLTests.kt

```diff
@@ -312,6 +312,60 @@ class PartiQLParserDDLTests {
                     ),
                 )
             ),
+
+            SuccessTestCase(
+                "CREATE TABLE with CASE SENSITIVE Identifier as column name",
+                """
```

Comment:
Could include a few more tests:
- struct with no attributes
- struct with multiple attributes
- struct with a field that has constraints
- array with no element type
- nested cases
  - array with a struct element type
  - struct type with an array field type